tests/test_audio_data.py

```diff
@@ -0,0 +1,66 @@
+import pytest
+import subprocess
+
+from importlib.resources import as_file, files
+from pathlib import Path
+from dragtor.audio_loader import AudioLoader
+from dragtor import config
+
+def test_ffmpeg_availability():
+    """Test if your local machine already installed with ffmpeg."""
+    try:
+        result = subprocess.run(["ffmpeg", "-version"], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+        assert result.returncode == 0
+    except FileNotFoundError:
+        pytest.fail("ffmpeg is not installed or not available in the system's PATH")
```

Comment:
This is a good idea! I think it would be great to have the same checks for 
- other external tools (llama-server + whisper)
- check that the defined model files are actually present

If you don't want to implement this, I can also do that

Reply:
I've some issue with llama-server implementation... Somehow it can be run through my terminal, but when run the module test, it can't find llama-server command